ballerina-tests/Dependencies.toml

```diff
@@ -105,7 +105,8 @@ dependencies = [
 	{org = "ballerina", name = "websocket"}
 ]
 modules = [
-	{org = "ballerina", packageName = "graphql_tests", moduleName = "graphql_tests"}
+	{org = "ballerina", packageName = "graphql_tests", moduleName = "graphql_tests"},
+	{org = "ballerina", packageName = "graphql_tests", moduleName = "graphql_tests.graphql"}
```

Comment:
Why is there a new submodule in the graphql_tests package?

Reply:
Seems there a graphql folder crated inside the graphql_tests package. Removed now